setup_build_environment.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+apt-get update
+apt-get install -y curl sudo build-essential vim openssh-client npm gcc g++ make nodejs
```

Comment:
I think most of these are already on the agent. Have you tried running without it?

Reply:
I rather for the install the requirements of the build rather than rely what is on the agent.